node/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "log4js": "~0.6.9",
     "bower": "~1.3.12",
-    "lodash": "~2.2.1"
+    "lodash": "~2.2.1",
+    "bower-config": "^0.5.2"
```

Comment:
Yes you right, I forgot that it was used for that... but it was added in previous commit, I don't know why you have to added it if it was there before.